docs/content/concepts/sui-architecture/consensus.mdx

```diff
@@ -1,109 +1,16 @@
 ---
-title: Sui Mempool and Consensus Engines
-sidebar_label: Mempool and Consensus Engines 
-description: Overview of the Sui mempool and consensus engines, Narwhal and Bullshark.
+title: Consensus
+sidebar_label: Consensus 
+description: Overview of the Sui consensus.
 ---
 
-This is a brief introduction to [Narwhal](https://github.com/MystenLabs/sui/tree/main/narwhal), and [Bullshark](https://arxiv.org/abs/2209.05633), the high-throughput mempool and consensus engines offered by Mysten Labs. Sui uses Narwhal as the mempool and Bullshark as the consensus engine by default, to sequence transactions that require a total ordering, synchronize transactions between validators and periodically checkpoint the network's state.
+## Overview
 
-The names highlight that the components split the responsibilities of:
+The basic purpose of consensus in blockchains is to agree on the order and ensure the availability of transactions.
+Validator operators and users can think of Sui consensus as a black box: validators submit different user transactions to consensus concurrently, and the consensus outputs a consistent stream of transactions across all well-behaving validators.
 
-- ensuring the availability of data submitted to consensus = [Narwhal](https://arxiv.org/abs/2105.11827)
-- agreeing on a specific ordering of this data = [Bullshark](https://dl.acm.org/doi/abs/10.1145/3548606.3559361)
+Byzantine-fault tolerant (BFT) consensus protocols are a fascinating area of research. If you want a deeper understanding of the consensus black box, the next-generation consensus engine set to roll out in Sui is based on the **Mysticeti** protocol. It represents the cutting edge of research, achieving the optimal frontier of low latency and high throughput by:
+- Weaving blocks into directed acyclic graphs (DAGs), utilizing the full bandwidth of the network and providing censorship resistance.
+- Mysticeti DAGs in particular are uncertified, where it takes only three rounds of messages to commit blocks from the DAGs.
```

Comment:
I believe we can also mention pipelining & multi leader here as it's quite powerful and allow us to commit on every round. If we don't want to be too specific on pipelining maybe we can just refer to the universal commit rule - I believe we don't use that term any more but  something like:

`Applying a novel commit rule allowing multiple leaders to commit per round and on every round`

Reply:
Added two points related to commit rule.